### ISSUE TITLE
Problem with conda installing

### ISSUE THREAD

**Issue by valery-shap** [rpetit3/dragonflye#12]
Hello, Robert,

Thank you for a great tool!
I had dragonflye 1.0.10 and now I decided to create new env with the new version. And I have problem the same as was with bactopia ( https://github.com/bactopia/bactopia/issues/334 ):
```
conda install -c bioconda dragonflye
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: \ 
```

and the process ends.
Are there any ideas how could I fix it? 

A lot of thank,
Valery

**Comment by rpetit3** [comment]
Hi @valery-shap,

Can you post the contents of `conda env export` while in your base environment? I'm also going to try and replicate locally on my end.

Cheers,
Robert

**Comment by valery-shap** [comment]
```
name: base
channels:
  - conda-forge
  - bioconda
  - defaults
dependencies:
  - _libgcc_mutex=0.1=conda_forge
  - _openmp_mutex=4.5=1_gnu
  - arrow=0.17.0=py38h578d9bd_1
  - attrs=20.3.0=pyhd3deb0d_0
  - backports=1.0=py_2
  - backports.functools_lru_cache=1.6.1=py_0
  - binaryornot=0.4.4=py_1
  - bzip2=1.0.8=h7f98852_4
  - c-ares=1.18.1=h7f98852_0
  - ca-certificates=2021.10.8=ha878542_0
  - certifi=2021.10.8=py38h578d9bd_1
  - cffi=1.14.0=py38he30daa8_1
  - chardet=3.0.4=py38_1003
  - click=7.1.2=pyh9f0ad1d_0
  - colorama=0.4.4=pyh9f0ad1d_0
  - commonmark=0.9.1=py_0
  - conda=4.11.0=py38h578d9bd_0
  - conda-package-handling=1.6.1=py38h7b6447c_0
  - cookiecutter=1.7.2=pyh9f0ad1d_0
  - coreutils=8.25=1
  - cryptography=2.9.2=py38h1ba5d50_0
  - curl=7.78.0=h1ccaba5_0
  - expat=2.2.10=he6710b0_2
  - future=0.18.2=py38h578d9bd_3
  - git=2.23.0=pl526hacde149_0
  - gitdb=4.0.5=pyhd8ed1ab_1
  - gitpython=3.1.13=pyhd8ed1ab_0
  - icu=68.2=h9c3ff4c_0
  - idna=2.9=py_1
  - importlib-metadata=3.4.0=py38h578d9bd_0
  - importlib_metadata=3.4.0=hd8ed1ab_0
  - jinja2=2.11.3=pyh44b312d_0
  - jinja2-time=0.2.0=py_2
  - jsonschema=3.2.0=py_2
  - krb5=1.19.2=hac12032_0
  - ld_impl_linux-64=2.33.1=h53a641e_7
  - libarchive=3.5.2=hccf745f_1
  - libcurl=7.78.0=h0b77cf5_0
  - libedit=3.1.20210910=h7f8727e_0
  - libev=4.33=h516909a_1
  - libffi=3.3=he6710b0_1
  - libgcc=7.2.0=h69d50b8_2
  - libgcc-ng=11.2.0=h1d223b6_11
  - libgomp=11.2.0=h1d223b6_11
  - libiconv=1.16=h516909a_0
  - libnghttp2=1.43.0=h812cca2_1
  - libsolv=0.7.19=h780b84a_5
  - libssh2=1.9.0=hab1572f_5
  - libstdcxx-ng=11.2.0=he4da1e4_11
  - libxml2=2.9.12=h72842e0_0
  - lz4-c=1.9.3=h9c3ff4c_1
  - lzo=2.10=h516909a_1000
  - mamba=0.15.3=py38h2aa5da1_0
  - markupsafe=1.1.1=py38h8df0ef7_2
  - ncurses=6.3=h7f8727e_2
  - nextflow=20.10.0=hecda079_0
  - nf-core=1.12.1=pyh3252c3a_0
  - openjdk=10.0.2=h14c3975_1015
  - openssl=1.1.1l=h7f98852_0
  - pcre=8.44=he1b5a44_0
  - perl=5.32.0=h36c2ea0_0
  - pip=20.0.2=py38_3
  - poyo=0.5.0=py_0
  - prompt-toolkit=3.0.16=pyha770c72_0
  - prompt_toolkit=3.0.16=hd8ed1ab_0
  - pycosat=0.6.3=py38h7b6447c_1
  - pycparser=2.20=py_0
  - pygments=2.8.0=pyhd8ed1ab_0
  - pyopenssl=19.1.0=py38_0
  - pyrsistent=0.17.3=py38h25fe258_1
  - pysocks=1.7.1=py38_0
  - python=3.8.3=hcff3b4d_0
  - python-dateutil=2.8.1=py_0
  - python-slugify=4.0.1=pyh9f0ad1d_0
  - python_abi=3.8=1_cp38
  - pyyaml=5.1.2=py38h516909a_0
  - questionary=1.9.0=pyhd8ed1ab_0
  - readline=8.0=h7b6447c_0
  - reproc=14.2.3=h7f98852_0
  - reproc-cpp=14.2.3=h9c3ff4c_0
  - requests=2.23.0=py38_0
  - requests-cache=0.5.0=py_0
  - rich=9.11.1=py38h578d9bd_0
  - ruamel_yaml=0.15.87=py38h7b6447c_0
  - setuptools=46.4.0=py38_0
  - six=1.14.0=py38_0
  - smmap=3.0.5=pyh44b312d_0
  - sqlite=3.31.1=h62c20be_1
  - tabulate=0.8.9=pyhd8ed1ab_0
  - text-unidecode=1.3=py_0
  - tk=8.6.8=hbc83047_0
  - tqdm=4.46.0=py_0
  - typing_extensions=3.7.4.3=py_0
  - unidecode=1.2.0=pyhd8ed1ab_0
  - urllib3=1.25.8=py38_0
  - wcwidth=0.2.5=pyh9f0ad1d_2
  - wheel=0.34.2=py38_0
  - whichcraft=0.6.1=py_0
  - xz=5.2.5=h7b6447c_0
  - yaml=0.1.7=had09818_2
  - zipp=3.4.0=py_0
  - zlib=1.2.11=h7b6447c_3
  - zstd=1.5.0=ha95c52a_0
prefix: /home/vshapovalova/miniconda3
```

**Comment by rpetit3** [comment]
Thank you! Going to see if I can replicate on my end!

**Comment by rpetit3** [comment]
Good news! I was able to replicate (also apologies on the delay getting to this). I gave up after ~10 minutes of solving

```
conda install -c conda-forge -c bioconda dragonflye
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: / 
```

I retried with Mamba, 

```
mamba install -c conda-forge -c bioconda dragonflye
```

And ran into this issue: 
```
Pinned packages:
  - python 3.9.*


Encountered problems while solving:
  - package dragonflye-1.0.6-hdfd78af_0 requires medaka >=1.4.4, but none of the providers can be installed
```

I'm going to review the dependencies. Looks like the issue is one updated to require `>3.9.*`, while medaka requires `<3.9`.

Will update sooner this time!

**Comment by rpetit3** [comment]
Interesting development, I'm unable to install dragonflye using Mamba v0.15.3, but can do so with Mamba v0.24.0

```
conda install -c conda-forge -c bioconda 'mamba>=0.24.0'
```

```
 mamba create -n test-dragonflye -c conda-forge -c bioconda dragonflye

                  __    __    __    __
                 /  \  /  \  /  \  /  \
                /    \/    \/    \/    \
███████████████/  /██/  /██/  /██/  /████████████████████████
              /  / \   / \   / \   / \  \____
             /  /   \_/   \_/   \_/   \    o \__,
            / _/                       \_____/  `
            |/
        ███╗   ███╗ █████╗ ███╗   ███╗██████╗  █████╗
        ████╗ ████║██╔══██╗████╗ ████║██╔══██╗██╔══██╗
        ██╔████╔██║███████║██╔████╔██║██████╔╝███████║
        ██║╚██╔╝██║██╔══██║██║╚██╔╝██║██╔══██╗██╔══██║
        ██║ ╚═╝ ██║██║  ██║██║ ╚═╝ ██║██████╔╝██║  ██║
        ╚═╝     ╚═╝╚═╝  ╚═╝╚═╝     ╚═╝╚═════╝ ╚═╝  ╚═╝

        mamba (0.24.0) supported by @QuantStack

        GitHub:  https://github.com/mamba-org/mamba
        Twitter: https://twitter.com/QuantStack

█████████████████████████████████████████████████████████████


Looking for: ['dragonflye']

bioconda/noarch                                      3.7MB @   3.0MB/s  1.3s
pkgs/main/linux-64                                   4.8MB @   3.5MB/s  1.5s
pkgs/r/noarch                                                 No change
pkgs/main/noarch                                              No change
pkgs/r/linux-64                                               No change
bioconda/linux-64                                    4.3MB @   1.9MB/s  2.3s
conda-forge/noarch                                   9.3MB @   3.1MB/s  3.1s
conda-forge/linux-64                                25.4MB @   3.2MB/s  8.3s
Transaction

  Prefix: /home/robert_petit/miniconda3/envs/test-dragonflye

  Updating specs:

   - dragonflye


... TRUNCATED ...

  Summary:

  Install: 205 packages

  Total download: 1GB

──────────────────────────────────────────────────────────────────────────────────────────────────────

Confirm changes: [Y/n]
```

**Comment by valery-shap** [comment]
Thank you for reply! I have mamba (0.15.3) and I was able to install bactopia using it. (conda didn't work for it too)
It is not about the topic this issue, but I noticed that dragonflye threw away small plasmids like ColRNA(less than 10kb) more often than Unicycler pipeline with long reads only. I will check during what step it happens, so maybe this difference is only because of different performance of flye or miniasm assemblers. And sometimes Unicycler is not able to assemble the whole chromosome unlike Flye. If it is so, the ideal receipt of using dragonflye tool could be to have 3 different assemblies with flye, miniasm and raven. And choose the better one.

Best regards,
Valery
